reproducer.yml

```diff
@@ -55,3 +55,9 @@
   roles:
     - role: ci_setup
     - role: reproducer
+
+  post_tasks:
+    - name: Allow traffic from OSP VMs to OSP API (needed for shiftstack)
+      ansible.builtin.shell: iptables -I LIBVIRT_FWI 1 -o ocpbm -j ACCEPT
+      become: true
+      when: cifmw_allow_vms_to_reach_osp_api
```

Comment:
This is needed for allowing the VMs to contact the OSP API, which is a requirement for the shiftstack OCP nodes to manage OSP resources that are created during the installation process and the openstack integration operators.